<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanMessageCreator</class>
 <widget class="QDialog" name="CanMessageCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Can Message Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Definition</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2,1">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>30</number>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Id:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Id format:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Size:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tx method:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Cycle time:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Main transmitter: </string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Coment:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_Name"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_Id"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_IdFormat">
           <item>
            <property name="text">
             <string>CAN Standard</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CAN Extended</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CAN FD Standard</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CAN FD Extended</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_Size"/>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox_TxMethod">
           <item>
            <property name="text">
             <string>NoMsgSendType</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cyclic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IfActive</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cyclicAndSpontanWithDelay</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>not_used</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_CycleTime"/>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBox_MainTransmitter"/>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit_Comment"/>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CanMessageCreator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CanMessageCreator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
